Sources/ApiVideoPlayer/SwiftUI/SwiftUIPlayerViewController.swift

```diff
@@ -47,6 +48,18 @@ public class SwiftUIPlayerViewController: UIViewController {
         super.viewDidDisappear(animated)
     }
 
+    override public var canBecomeFirstResponder: Bool {
+        true
+    }
+
+    override public func remoteControlReceived(with event: UIEvent?) {
+        if let event = event {
+            if event.type == .remoteControl {
+                self.playerView.remoteControlEventReceived(with: event)
```

Comment:
COnsistency: You should rename `remoteControlEventReceived` to `remoteControlReceived`